src/components/artifacts/viewer.tsx

```
@@ -182,43 +196,82 @@ const CodeRenderer = ({
+        {content}
+      </SyntaxHighlighter>
+    </div>
   );
 };
 
 const SVGRenderer = ({ content }: { content: string }) => {
-  const preview = (
-    <div className="border rounded-lg p-4 bg-background flex items-center justify-center">
+  return (
+    <div className="border p-4 bg-background flex items-center justify-center">
       <div dangerouslySetInnerHTML={{ __html: content }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)